frontend/src/components/elements/Text/Text.tsx

```diff
@@ -50,6 +55,19 @@ const linkWithTargetBlank = (props: LinkProps): ReactElement => (
   </a>
 )
 
+const linkReferenceHasParens = (props: LinkReferenceProps): any => {
+  console.log(props)
```

Comment:
Remove `console.log`. If you really want to log this for future debugging, use `logMessage` from `lib/log`